package.json

```diff
@@ -158,13 +158,13 @@
         "category": "PowerShell"
       },
       {
-        "command": "PowerShell.EnableISEMode",
-        "title": "Enable ISE Mode",
+        "command": "PowerShell.EnterISEMode",
+        "title": "Enter ISE Mode",
         "category": "PowerShell"
       },
       {
-        "command": "PowerShell.DisableISEMode",
-        "title": "Disable ISE Mode (restore to defaults)",
+        "command": "PowerShell.ExitISEMode",
+        "title": "Exit ISE Mode (restore to defaults)",
```

Comment:
How much will this affect:

- existing expectations
- documentation

Also, enable/disable carries more permanence than enter/exit. Does ISE mode persist?

Reply:
yeah it does. Since it's just enabling settings. I guess we can go with enable/disable Notebook Mode?

Reply:
I'm happy with enable/disable. It's the presence of `Notebook` that's important to me, since it's what I'm searching for:

- Something that comes up in the command palette when looking for `Notebook` to start and stop notebook mode
- A button that conspicuously gets me out of notebook mode

Reply:
I'll address this along with the breaking change in a different PR then.